cvat/apps/dataset_manager/annotation.py

```diff
@@ -502,7 +500,9 @@ def _modify_unmached_object(self, obj, end_frame):
                 self._modify_unmached_object(element, end_frame)
 
     @staticmethod
-    def get_interpolated_shapes(track, start_frame, end_frame, dimension):
+    def get_interpolated_shapes(
+        track, start_frame, end_frame, dimension, is_skeleton=False
```

Comment:
I would suggest something like this:

```suggestion
        track, start_frame, end_frame, dimension, *, include_outside_frames=False
```

so that the name of the parameter actually describes what it does.